### PR TITLE
Wazuh 4.4.0 release notes and versions update

### PR DESCRIPTION
**Pull request by Arcapas**
## Description
This issue aims to create the Wazuh v4.4 release notes and versions update throughout the documentation.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/release-notes/release-4-4-0.rst [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+This section lists the changes in version 4.4.0. Every update of the Wazuh solution is cumulative and includes all enhancements and fixes from previous releases.
+
+Highlights
+----------
+
+This new version of Wazuh brings new features and adds support related to different Linux distributions and integrations. For more details, the highlights of Wazuh 4.4.0 are listed below:
+
+- IPv6 support for the enrollment process and the agent-manager connection
+- Vulnerabilities detection support for SUSE agents
+- Expanded operating systems support with Alpine Linux agent
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:18]:
```suggestion
- Alpine Linux agent to expand the operating systems supported
```

File: source/release-notes/release-4-4-0.rst [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+- Wazuh indexer and dashboard based on the latest Opensearch 2.4.1 version
+- Rework of Ubuntu Linux 20.04 and 22.04 SCA policies
+- Support for Azure Integration in Linux agents
+
+Below you can find more information about each of these highlights.
+
+Wazuh 4.4.0 brings IPv6 support when connecting and enrolling an agent to a manager. The IPv6 protocol can handle packets more effectively, enhance performance, and boost security. This new feature allows agents to register and connect through an IPV6 address.
+
+SUSE agents now natively support vulnerabilities detection. Wazuh added full support for SUSE Linux Enterprise Server and Desktop 11, 12, and 15 operating systems. The vulnerability Detector now scans the programs identified by ``syscollector``, looking to report vulnerabilities described in the `SUSE OVAL <https://www.suse.com/support/security/oval/>`_ and the `NVD <https://nvd.nist.gov/>`_ databases.
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:27]:
```suggestion
SUSE agents now natively support vulnerabilities detection. Wazuh added full support for SUSE Linux Enterprise Server and Desktop operating systems versions 11, 12, and 15. The vulnerability Detector now scans the programs identified by ``syscollector``, looking to report vulnerabilities described in the `SUSE OVAL <https://www.suse.com/support/security/oval/>`_ and the `NVD <https://nvd.nist.gov/>`_ databases.
```

File: source/release-notes/release-4-4-0.rst [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+
+Wazuh 4.4.0 adds support for Alpine Linux. As Alpine Linux is a distribution that has gained great popularity among Linux users, we saw the need for the Wazuh agent to support Alpine Linux and ensure the proper support for this distribution. We generated Alpine Linux installers to make it easier to install and thus spare users of this distribution from having to compile the source code locally.
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:29]:
```suggestion
Wazuh 4.4.0 adds support for Alpine Linux. As Alpine Linux is a distribution that has gained great popularity among Linux users, we saw the need for the Wazuh agent to support Alpine Linux and ensure the proper support for this distribution. We generated Alpine Linux installers to make it easier to install and thus spare users from having to compile the source code locally.
```

File: source/release-notes/release-4-4-0.rst [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+
+.. thumbnail:: /images/release-notes/4.4.0/alpine-linux.png
+      :align: center
+      :width: 60%
+      :title: This new version adds support for Alpine Linux
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:34]:
```suggestion
      :title: Wazuh 4.4.0 adds support for Alpine Linux
```

File: source/release-notes/release-4-4-0.rst [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+
+Wazuh indexer and dashboard bump to OpenSearch 2.4.1. The Wazuh indexer and the Wazuh dashboard are based on OpenSearch, an open source search and analytics project derived from Elasticsearch and Kibana. We generated and tested the ``wazuh-indexer`` Debian and RPM packages with OpenSearch 2.4.1 and the ``wazuh-dashboard`` Debian and RPM packages with OpenSearch dashboards 2.4.1. This way, we avoid earlier version vulnerabilities, have all OpenSearch new functionalities, and are up to date with the OpenSearch project development.
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:36]:
```suggestion
Wazuh indexer and dashboard based on the latest OpenSearch 2.4.1. The Wazuh indexer and the Wazuh dashboard are based on OpenSearch, an open source search and analytics project derived from Elasticsearch and Kibana. We generated and tested the ``wazuh-indexer`` Debian and RPM packages with OpenSearch 2.4.1 and the ``wazuh-dashboard`` Debian and RPM packages with OpenSearch dashboards 2.4.1. With this bump to the latest version, we avoid earlier version vulnerabilities, have all OpenSearch new functionalities, and are up to date with the OpenSearch project development.
```

File: source/release-notes/release-4-4-0.rst [hunk 6]
```diff
@@ -0,0 +1,505 @@
...
+
+To solve some errors in the previous Ubuntu Linux 20.04 SCA Policy, we reworked the Ubuntu Linux 20.04 and 22.04 SCA policies. As part of this task, we used the *CIS Ubuntu Linux 22.04 LTS Benchmark v1.0.0* to update Ubuntu Linux 22.04 SCA Policy.
+
+Wazuh added support for Azure Integration in Linux agents. Now this integration can run for both agents and managers. We modified the packages generation process to support Azure in those agents installed using the WPK packages. Each new WPK package contains all the updated binaries and source code, and the installer updates all files and binaries to support Azure integration.
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:40]:
```suggestion
Wazuh added support for Azure Integration in Linux agents. Now this integration can run for both agents and managers. We modified the packages generation process to support Azure in those agents that are installed using the WPK packages. Each new WPK package contains all the updated binaries and source code, and the installer updates all files and binaries to support Azure integration.
```

File: source/release-notes/release-4-4-0.rst [hunk 7]
```diff
@@ -0,0 +1,505 @@
...
+
+.. thumbnail:: /images/release-notes/4.4.0/azure-integration.png
+      :align: center
+      :width: 60%
+      :title: This new version adds support Azure Integration in Linux agents
```
Comment (javimed) [inline source/release-notes/release-4-4-0.rst:45]:
```suggestion
      :title: Wazuh 4.4.0 adds support for the Azure Integration in Linux agents
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/_static/js/redirects.js [hunk 8]
```diff
@@ -170,6 +170,7 @@ newUrls['4.3'] = [
   '/release-notes/release-4-3-8.html',
   '/release-notes/release-4-3-9.html',
   '/release-notes/release-4-3-10.html',
+  '/release-notes/release-4-4-0.html',
```
[the remark this answers is not in the document]
Comment (Arcapas) [reply source/_static/js/redirects.js:173]: Yes, I'll do it.

**Pull request merged**